src/core/Dropdown/Dropdown/Dropdown.tsx

```diff
@@ -241,37 +164,66 @@ export class Dropdown extends Component<DropdownProps> {
         id={id}
         {...passProps}
       >
-        <HtmlLabel
-          id={labelId}
-          {...labelProps}
-          className={dropdownClassNames.label}
-        >
-          {labelMode === 'hidden' ? (
-            <VisuallyHidden>{labelText}</VisuallyHidden>
-          ) : (
-            <Paragraph {...labelTextProps}>{labelText}</Paragraph>
-          )}
-        </HtmlLabel>
-        <ListboxInput {...listboxInputProps}>
-          <ListboxButton {...passDropdownButtonProps}>
-            {listboxDisplayValue}
-          </ListboxButton>
-          {!!ListboxPopoverComponentReplace ? (
-            <ListboxPopoverComponentReplace {...passDropdownPopoverProps}>
-              {this.dropdownItems(children, passDropdownItemProps)}
-            </ListboxPopoverComponentReplace>
-          ) : (
+        <HtmlDiv>
+          <LabelText
+            id={labelId}
+            labelMode={labelMode}
+            as="label"
+            optionalText={optionalText}
+          >
+            {labelText}
+          </LabelText>
+          <ListboxInput
+            aria-labelledby={ariaLabelledByIds}
+            disabled={disabled}
+            onChange={onChange}
+            name={name}
+            value={selectedValue || ''}
+          >
+            <ListboxButton
+              {...dropdownButtonProps}
+              id={buttonId}
+              className={classnames(dropdownClassNames.button, {
+                [dropdownClassNames.disabled]: !!disabled,
+              })}
+              {...getConditionalAriaProp('aria-labelledby', [
+                selectedValue === undefined ? ariaLabelledByIds : undefined,
+              ])}
```

Comment:
I think we could also have it like this and not using the const with string interpolation:

```tsx
{...getConditionalAriaProp(
  'aria-labelledby',
  selectedValue === undefined ? [labelId, ariaLabelledBy] : [],
)}
```

Would this be a little more readable?

Reply:
The first could also probably be `selectedValue === undefined ? [labelId, ariaLabelledBy] : undefined,` and if selectedValue is always either truthy or not truthy (how do we indicate no selection yet?) then `!!selectedValue ? [labelId, ariaLabelledBy] : undefined`

Reply:
Yeah, maybe it's better to rely on the util in all cases. I'd stick with the `selectedValue === undefined` to be on the safe side since the values can vary.

Reply:
Should be ok without truthy checking as there can be empty strings in some cases.